hack/observability/opentelemetry/chart/Chart.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v2
+appVersion: 0.22.0
+description: OpenTelemetry Collector Helm chart for Kubernetes
+home: https://opentelemetry.io/
+icon: https://opentelemetry.io/img/logos/opentelemetry-logo-nav.png
+maintainers:
+- name: bogdandrutu
```

Comment:
is having the maintainers of a Helm chart in the repo common practice?

Reply:
I think it's recommended to have maintainers in the Chart.yaml.

What we did here is fetch the chart and store it as a set of local files, to avoid installing a tilt addon for fetching remote charts, to avoid fetching things over the internet, and to pin at a certain chart version.